### PR TITLE
Signup updates

### PR DESCRIPTION
**Pull request by kschzt**

### CODE REVIEW

File: browser/scripts/account-controller.js [hunk 1]
```diff
@@ -96,21 +96,20 @@ AccountController.prototype._bindEvents = function(el, dfd)
 			currentUsername = E2.models.user.get('username');
 		}
 		if (($(this).attr('name') === 'username') && $(this).val() && $(this).val()!==currentUsername) {
+			return
```
Comment (kkananen) [inline browser/scripts/account-controller.js]: is this meant to be here?

Comment (kschzt) [inline browser/scripts/account-controller.js]: @imgrby

File: browser/scripts/account-controller.js [hunk 2]
```diff
@@ -390,50 +401,76 @@ AccountController.prototype.openResetPasswordModal = function(dfd) {
...
+
 AccountController.prototype.openAccountModal = function(dfd) {
 	var that = this;
 	var dfd = dfd || when.defer();
 	var accountTemplate = E2.views.account.account;
 	
-	ga('send', 'event', 'account', 'open', 'resetModal');
-	
+	//ga('send', 'event', 'account', 'open', 'accountModal');
+	console.trace();
```
Comment (gmarinov) [inline browser/scripts/account-controller.js]: remember to clean up..

File: browser/scripts/account-controller.js [hunk 3]
```diff
@@ -390,50 +401,76 @@ AccountController.prototype.openResetPasswordModal = function(dfd) {
...
+		onEscape: function() {
+			dfd.resolve();
+		}
 	}).init(function() {
 		E2.app.useCustomBootboxTemplate(accountTemplate);
+		that.fillAccountForm();
 	});
 
-	this._bindEvents(bb, dfd);
+	//this._bindEvents(bb, dfd);
```
Comment (gmarinov) [inline browser/scripts/account-controller.js]: so, with or without this here?

File: browser/scripts/account-controller.js [hunk 4]
```diff
@@ -390,50 +401,76 @@ AccountController.prototype.openResetPasswordModal = function(dfd) {
...
+				$('#signup-form_id .required').parent().addClass('wrong');
 			},
 			success: function(user)
 			{
-				ga('send', 'event', 'account', 'accountUpdated', user.username)
+				ga('send', 'event', 'account', 'accountUpdated', user.username);
 				bootbox.hideAll();
-				bootbox.alert('Account updated!');
-				dfd.resolve();
+				window.location.reload();
```
Comment (gmarinov) [inline browser/scripts/account-controller.js]: if this is just updating the account details then just update the user in environment
only when registering a new user, then reload the entire page. (for now)

File: controllers/userController.js [hunk 5]
```diff
@@ -75,7 +75,7 @@ function parseErrors(errors) {
  */
  exports.logout = function(req, res) {
  	req.logout();
- 	res.redirect('/login');
+ 	res.redirect('/edit');
```
Comment (gmarinov) [inline controllers/userController.js]: i'd go with just '/' here. it has a nice big link to /edit there anyway.

File: browser/scripts/account-controller.js (hunk 4 above)
Comment (imgrby) [inline browser/scripts/account-controller.js]: right

File: browser/scripts/account-controller.js (hunk 3 above)
Comment (imgrby) [inline browser/scripts/account-controller.js]: fixed

File: browser/scripts/account-controller.js (hunk 2 above)
Comment (imgrby) [inline browser/scripts/account-controller.js]: fixed

File: controllers/userController.js (hunk 5 above)
Comment (imgrby) [inline controllers/userController.js]: fixed

**Pull request merged**